### ISSUE TITLE
NPM Run commands throw errors

### ISSUE THREAD

**Issue by aaronosher** [VATUSA/frontend#7]
```
$ npm run start

> vatusa-frontend@0.1.0 start /VATUSA/frontend
> node scripts/start.js

/VATUSA/frontend/scripts/start.js:66
    );
    ^
SyntaxError: Unexpected token )
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vatusa-frontend@0.1.0 start: `node scripts/start.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vatusa-frontend@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!    ~/.npm/_logs/2017-08-29T03_06_03_593Z-debug.log
```
```
$ npm run build

> vatusa-frontend@0.1.0 build /VATUSA/frontend
> node scripts/build.js

/VATUSA/frontend/scripts/build.js:72
          ),
          ^
SyntaxError: Unexpected token )
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vatusa-frontend@0.1.0 build: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vatusa-frontend@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2017-08-29T03_01_12_543Z-debug.log
```
```
$ npm run test

> vatusa-frontend@0.1.0 test /VATUSA/frontend
> node scripts/test.js --env=jsdom

2017-08-29 04:03 node[86606] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2017-08-29 04:03 node[86606] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: Error watching file for changes: EMFILE
    at exports._errnoException (util.js:1022:11)
    at FSEvent.FSWatcher._handle.onchange (fs.js:1406:11)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vatusa-frontend@0.1.0 test: `node scripts/test.js --env=jsdom`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vatusa-frontend@0.1.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2017-08-29T03_03_53_366Z-debug.log
```

**Comment by aaronosher** [comment]
The issue appears to be local to NPM on Mac, caused by the trailing commas needed by Travis-CI.

**Comment by rahulp959** [comment]
Node 8 is what we currently use, please ensure you're up to date :)

**Comment by rahulp959** [comment]
However it would indicate a larger problem, I'll pull and check, also feel free to tag me in these so I get a notification / ping me on slack

**Comment by rahulp959** [comment]
@aaronosher

**Comment by aaronosher** [comment]
Resolved #8
